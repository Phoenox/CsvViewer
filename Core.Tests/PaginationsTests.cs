namespace Core.Tests;

public class PaginationsTests
{
	[Fact]
	public void IntegationTest()
	{
		var file = Dummies.CsvFile;
		var page = Pagination.GetPage(file, 17, 5);
		page.Should().MatchSnapshot();
	}
}
