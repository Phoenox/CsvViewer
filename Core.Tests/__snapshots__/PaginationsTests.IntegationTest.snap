{
  "Headers": [
    "foo",
    "bar",
    "bla"
  ],
  "Lines": [
    {
      "Cells": [
        "85",
        "85",
        "85"
      ]
    },
    {
      "Cells": [
        "86",
        "86",
        "86"
      ]
    },
    {
      "Cells": [
        "87",
        "87",
        "87"
      ]
    },
    {
      "Cells": [
        "88",
        "88",
        "88"
      ]
    },
    {
      "Cells": [
        "89",
        "89",
        "89"
      ]
    }
  ],
  "TotalLines": 100
}
